### ISSUE TITLE
Add a link to the github repo on the splash page at the bottom

### ISSUE THREAD

**Issue by austintackaberry** [austintackaberry/ydkjs-exercises#34]
Users should be able to easily access the source code, so that they can look if they're curious or contribute if they are interested. Maybe something like this:

"This is an active open source project. If you want to take a look at the source code or contribute to the project, then you can check out the repository here: https://github.com/austintackaberry/ydkjs-exercises"

**Comment by IrritatedEllipses** [comment]
Seems like a good first one for me. I'll get on it.

**Comment by austintackaberry** [comment]
It's yours :slightly_smiling_face:

**Comment by IrritatedEllipses** [comment]
Since I'm going to be putting that in anyway, is there a license that we need to be adding as well?

**Comment by austintackaberry** [comment]
I don't think that the license needs to be added to the website.

I think the splash page is a good start. I do like the idea of having it on every page...Perhaps, it could be added to a footer of some sort that would show on every page?

**Comment by IrritatedEllipses** [comment]
That was my plan so I wrote out a footer that just displayed as the final component on App.js.

Seems to be the way to do it, right now it's just a simple text line but having it in there can lead to navigation / information being displayed / what have you in the future. I also changed the wording a bit: 

> YDKJS Exercises is created and maintained by learners like you! To take a look at the source code or contribute to the project, check out the repository at https://github.com/austintackaberry/ydkjs-exercises

But I have no emotional investment in it. It does, however, more closely match the accepting way you phrased your first reddit post.
